### PR TITLE
Address SSL certificate issues around proxy

### PR DESCRIPTION
**Pull request by abdulmujeeb**
This adds steps to get DevSecOps Studio running behind corporate proxies which does MITM.

### CODE REVIEW

**Comment by abdulmujeeb** [comment]
Document for SSL Certificate issues over proxy

File: docs/troubleshooting.md [hunk 1]
```diff
@@ -33,3 +33,46 @@ The following snippet is shows that jenkins role is being executed at the moment
...
+Follow the similar procedure for vagrant command 
+
+1. Go to Vagant home (/opt/vagrant)
+
+2. Make a backup of cacert.pem
+
+3. Open cacert.pem in any text editor and copy the conents of corporate_root_certificate.pem file at the end of cacert.pem file. Save the file.
+
+   The content of corporate_root_certificate.pem file would look something like 
+
```
Comment (secfigo) [inline docs/troubleshooting.md]: Add formatting for text, more details - https://help.github.com/articles/creating-and-highlighting-code-blocks/

File: docs/troubleshooting.md [hunk 2]
```diff
@@ -33,3 +33,46 @@ The following snippet is shows that jenkins role is being executed at the moment
...
+and command execution fails. The error is caused because the commands usually uses OpenSSL internally which doesn't recongnize the corporate 
+root certificate
+
+# Certificate validation error when running ansible-galaxy command
+
+1. Acccess any https website from your browser (Chrome) and download the certificate by clicking on the lock icon in the address bar and save it as .pem file (corporate_root_certificate.pem)
+
+2. Run following command to identify the path of openssl cacerts locations (its usuall /usr/local/etc/openssl/certs)
+
+> brew info openssl
```
Comment (secfigo) [inline docs/troubleshooting.md]: put it under 
```bash  
$ brew info openssl
```

File: docs/troubleshooting.md [hunk 3]
```diff
@@ -33,3 +33,46 @@ The following snippet is shows that jenkins role is being executed at the moment
...
+
+2. Run following command to identify the path of openssl cacerts locations (its usuall /usr/local/etc/openssl/certs)
+
+> brew info openssl 
+
+3. Copy corporate_root_certificate.pem in /usr/local/etc/openssl/certs folder
+
+4. Run the following command to include the corporate root certificate in openssl 
+
+> /usr/local/opt/openssl/bin/c_rehash
```
Comment (secfigo) [inline docs/troubleshooting.md]: same as above

File: docs/troubleshooting.md [hunk 4]
```diff
@@ -33,3 +33,46 @@ The following snippet is shows that jenkins role is being executed at the moment
...
+
+## Running the commands behind corporate proxy 
+
+Certificate validation failed errors
+
+Generally running the ansible-galaxy and vagrant behind the corporate proxy would throw Certificate Validation failed error 
+and command execution fails. The error is caused because the commands usually uses OpenSSL internally which doesn't recongnize the corporate 
+root certificate
+
+# Certificate validation error when running ansible-galaxy command
```
Comment (secfigo) [inline docs/troubleshooting.md]: put ### here instead of #

File: docs/troubleshooting.md (hunk 1 above)
Comment (abdulmujeeb) [reply docs/troubleshooting.md]: Completed the file beautification as per your review comments

**Comment by abdulmujeeb** [comment]
Completed the file beautification as per your review comments

**Comment by secfigo** [comment]
Thank you @abdulmujeeb for the PR :)

**Pull request merged**